es6/jsxcad-ui-threejs.js

```diff
@@ -619,7 +619,13 @@ const merge = async (properties, parameters) => {
 };
 
 const setMaterial = async (definitions, tags, parameters) => {
-  const threejsMaterial = toThreejsMaterialFromTags(tags, definitions);
+  let threejsMaterial;
+  if (tags.includes('type:ghost')) {
+    // FIX: This is ugly.
```

Comment:
How so?

Reply:
Well, it's just hacking in an arbitrary material if it's a ghost geometry. Actually, a material should probably be set on creation (and transparency set likewise)